### PR TITLE
Add reconcile_binary_keys to FoldTracedData

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: core_data_modules/traced_data/util/fold_traced_data.py [hunk 1]
```diff
@@ -255,6 +262,27 @@ def reconcile_yes_no_keys(cls, user, td_1, td_2, keys):
...
+
+        for key in keys:
+            if cls._is_missing_value(td_1.get(key)) and cls._is_missing_value(td_2.get(key)):
+                binary_dict[key] = cls.reconcile_missing_values(td_1.get(key), td_2.get(key))
+            elif cls._is_missing_value(td_1.get(key)):
+                binary_dict[key] = td_2.get(key)
+            elif cls._is_missing_value(td_2.get(key)):
+                binary_dict[key] = td_1.get(key)
+            elif td_1.get(key) == "ambivalent" or td_1.get(key) == "ambivalent":
+                binary_dict[key] = "ambivalent"
```
Comment (lukechurch) [inline core_data_modules/traced_data/util/fold_traced_data.py]: Please pull this string out into a constant

File: core_data_modules/traced_data/util/fold_traced_data.py [hunk 2]
```diff
@@ -255,6 +262,27 @@ def reconcile_yes_no_keys(cls, user, td_1, td_2, keys):
         td_1.append_data(yes_no_dict, Metadata(user, Metadata.get_call_location(), time.time()))
         td_2.append_data(yes_no_dict, Metadata(user, Metadata.get_call_location(), time.time()))
 
+    @classmethod
+    def reconcile_binary_keys(cls, user, td_1, td_2, keys):
```
Comment (lukechurch) [inline core_data_modules/traced_data/util/fold_traced_data.py]: I'm afraid I don't find this name self-explanatory - could you add a doc comment please

File: core_data_modules/traced_data/util/fold_traced_data.py (hunk 1 above)
Comment (as2388) [reply core_data_modules/traced_data/util/fold_traced_data.py]: done

File: core_data_modules/traced_data/util/fold_traced_data.py (hunk 2 above)
Comment (as2388) [reply core_data_modules/traced_data/util/fold_traced_data.py]: added

**Pull request merged**